packages/core/.storybook/main.ts

```diff
@@ -40,36 +35,40 @@ const getAddons = () => {
 
   return addons;
 };
-const config: StorybookConfig = {
+
+export default {
   stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
   addons: getAddons(),
   framework: {
-    name: "@storybook/react-webpack5",
+    name: "@storybook/react-vite",
     options: {}
   },
-  docs: {
-    autodocs: false
+  core: {
+    builder: "@storybook/builder-vite"
   },
   typescript: {
     check: true,
-    checkOptions: {
-      async: false
-    },
     reactDocgen: "react-docgen-typescript"
   },
-  async webpackFinal(config, { configType }) {
-    if (configType === "DEVELOPMENT") {
-      if (config.resolve) {
-        config.resolve.alias = {
-          ...config.resolve.alias,
+  staticDirs: ["./static"],
+  async viteFinal(config, { configType }) {
+    const { mergeConfig } = await import("vite");
+
+    return mergeConfig(config, {
+      resolve: {
+        alias: {
           "monday-ui-style/dist/index.min.css": path.resolve(__dirname, "../../style/src/index.scss"),
-          "monday-ui-style/dist": path.resolve(__dirname, "../../style/src")
-        };
+          "monday-ui-style/dist": path.resolve(__dirname, "../../style/src"),
+          // mixins workaround for vite:
+          "~monday-ui-style/dist/mixins": path.resolve(__dirname, "../../style/src/mixins"),
+          "~monday-ui-style/dist/functions": path.resolve(__dirname, "../../style/src/functions"),
+          "~monday-ui-style": path.resolve(__dirname, "../../style"),
```

Comment:
notice this would need changes once you merge the refactor of the tokens

Reply:
also, what's the tilde "~" sign stands for here? the two first lines in your `alias` config doesn't include it